awx/ui_next/src/screens/Inventory/InventorySources/InventorySourceListItem.test.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import InventorySourceListItem from './InventorySourceListItem';
+
+const source = {
+  id: 1,
+  name: 'Foo',
+  source: 'Source Bar',
+  summary_fields: { user_capabilities: { start: true, edit: true } },
+};
+describe('<InventorySourceListItem />', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('should mount properly', () => {
+    let wrapper;
+    const onSelect = jest.fn();
+    act(() => {
+      wrapper = mountWithContexts(
+        <InventorySourceListItem
+          source={source}
+          isSelected={false}
+          onSelect={onSelect}
+          label="Source Bar"
+        />
+      );
+    });
+    expect(wrapper.find('InventorySourceListItem').length).toBe(1);
+  });
+  test('all buttons and text fields should render properly', () => {
+    let wrapper;
+    const onSelect = jest.fn();
+    act(() => {
+      wrapper = mountWithContexts(
+        <InventorySourceListItem
+          source={source}
+          isSelected={false}
+          onSelect={onSelect}
+          label="Source Bar"
+        />
+      );
+    });
+    expect(wrapper.find('DataListCheck').length).toBe(1);
+    expect(
+      wrapper
+        .find('DataListCell')
+        .at(0)
+        .text()
+    ).toBe('Foo');
+    expect(
+      wrapper
+        .find('DataListCell')
+        .at(1)
+        .text()
+    ).toBe('Source Bar');
+    expect(wrapper.find('PencilAltIcon').length).toBe(1);
+  });
+
+  test('item should be checked', () => {
+    let wrapper;
+    const onSelect = jest.fn();
+    act(() => {
```

Comment:
I don't think we need `act()` to mount this component since it doesn't have effects or state updates.